brave/brave-core
# Hide first run dialog

fix https://github.com/brave/brave-browser/issues/27225

Temporarily hide till we have numbers from new welcome flow.
If welcome flow do better than FR dialog, we'll delete FR dialog.


Resolves 

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch Brave with clean profile
2. Check first run dialog is not launched
